django/db/backends/oracle/introspection.py

```diff
@@ -121,15 +121,18 @@ def get_table_description(self, cursor, table_name):
                     )
                     THEN 1
                     ELSE 0
-                END as is_json
+                END as is_json,
+                user_col_comments.comments as column_comment
             FROM user_tab_cols
             LEFT OUTER JOIN
                 user_tables ON user_tables.table_name = user_tab_cols.table_name
-            WHERE user_tab_cols.table_name = UPPER(%s)
-        """, [table_name])
+            LEFT OUTER JOIN
+                user_col_comments ON user_col_comments.column_name = user_tab_cols.column_name
+            WHERE user_tab_cols.table_name = UPPER(%s) and user_col_comments.table_name = UPPER(%s)
```

Comment:
You can `JOIN` with `user_col_comments` on multiple columns there is no need to use `UPPER()` again and pass `table_name` multiple times.

Reply:
I fixed this query to below

~~~SQL
...
                END as is_json,
                user_col_comments.comments as col_comment
            FROM user_tab_cols
            LEFT OUTER JOIN
                user_tables ON user_tables.table_name = user_tab_cols.table_name
            LEFT OUTER JOIN
                user_col_comments
                ON user_col_comments.column_name = user_tab_cols.column_name
                    and user_col_comments.table_name = %s
            WHERE user_tab_cols.table_name = %s

~~~